docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: '3.6'
```

Comment:
why was this deleted?

Reply:
After updating docker, I started getting a warning. Apparently, with the new docker update the version property is obsolete. When running docker compose as default it will always use the newest version. link: https://forums.docker.com/t/docker-compose-yml-version-is-obsolete/141313/2